### DISCUSSION TITLE
Unable to enable Wikis (org)

### DISCUSSION THREAD

**Discussion by NidalGhonaim** [community/community#20999] [Projects and Issues]
My organization upgraded to the paid plan that includes Wikis recently.
Now old repos that already existed don't have wikis. But ones created more recently do.
When I go to enable the Wikis (in an older repo) it still says "Upgrade or make this repo public.."

Not sure how to solve it or what to tell the org owners to do.

**Comment by NidalGhonaim** [comment]
Turns out the more recent repos are public and that's why the Wikis are available for them.

**Comment by vfalasconi** [comment]
I'm having the same issue with a GitHub Enterprise private repository: when I go to repository Settings > Features > Wikis, this message is displayed: "Upgrade or make this repository public to enable Wikis". I would like to use Wiki functionality, without having to publish my repository. We're a team and such a feature can be useful.
